### PR TITLE
Update docker tag generation mechanism 

### PR DESCRIPTION
**Pull request by finalspy**
+ update kordamp plugin version
missing : update metadata.yaml

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -77,10 +83,10 @@ config {
 object VersionInfo {
     const val kotlin = "1.3.70"
     const val jackson = "2.10.3"
-    const val kordamp = "0.32.0"
+    const val kordamp = "0.38.0"
```
Comment (PierreLeresteux) [inline build.gradle.kts:86]: remove ?

File: settings.gradle [hunk 2]
```diff
@@ -23,7 +23,7 @@ pluginManagement {
 }
 plugins {
     id("com.gradle.enterprise") version "3.0"
-    id("org.kordamp.gradle.settings") version "0.29.0"
+    id("org.kordamp.gradle.settings") version "0.38.0"
```
Comment (PierreLeresteux) [inline settings.gradle:26]: remove ?

File: build.gradle.kts [hunk 3]
```diff
@@ -24,11 +24,11 @@ plugins {
     id("com.gradle.plugin-publish") version "0.10.1"
     id("io.gitlab.arturbosch.detekt") version "1.3.1"
     id("org.jmailen.kotlinter") version "2.1.3"
-    id("org.kordamp.gradle.project") version "0.32.0"
+    id("org.kordamp.gradle.project") version "0.38.0"
     kotlin("jvm") version "1.3.70"
 }
 
-version = "1.1.3"
+version = "1.1.3-yann"
```
[the remark this answers is not in the document]
Comment (finalspy) [reply build.gradle.kts]: 1.2.0

File: build.gradle.kts (hunk 1 above)
Comment (finalspy) [reply build.gradle.kts:86]: no used later in the file

File: settings.gradle (hunk 2 above)
Comment (finalspy) [reply settings.gradle]: ok

File: src/main/kotlin/com/saagie/technologies/model/DockerInfo.kt [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+data class DockerInfo(
+    val image: String,
+    val baseTag: String,
+    val dynamicVersion: String? = null,
+    val version: String = "$baseTag-$dynamicVersion"
+) {
+    constructor() : this("", "")
+
+    fun generateDocker() = "$image:$baseTag-$dynamicVersion".removeIllegalDockerCharacters()
```
Comment (PierreLeresteux) [inline src/main/kotlin/com/saagie/technologies/model/DockerInfo.kt:28]: fun generateDocker() = "$image:$version".removeIllegalDockerCharacters()

**Pull request merged**